File: Beginner/Palindromic substrings (STRPALIN)/palindrome.py
t = int(input())

while t:
    A = input()
    B = input()
    f = False;
    for i in A:
        if i in B:
            f = True;
            break;
    if f == True:
        print("Yes")
    else:
        print("No")
        

    t = t-1

File: Beginner/Palindromic substrings (STRPALIN)/palindrome.cpp
#include<bits/stdc++.h>

using namespace std;
int main() {
    int t;
    cin>>t;
    string a,b;
    while(t--) {
        cin>>a>>b;
        bool f=false;
        for(int i=0;i<a.length();i++) {
            for(int j=0;j<b.length();j++) {
                if(a[i]==b[j])
                {
                    f=true;
                    break;
                }
            }
        }
        if(f)
          cout<<"Yes"<<endl;
          else
          cout<<"No"<<endl;
        
    }
    return 0;
}